src/components/Notifications/Notifications.tsx

```diff
@@ -93,27 +93,15 @@ export default function Notifications() {
         return null
       }
 
-      try {
-        return await Governance.get().getUserLastNotification()
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      } catch (error: any) {
-        if (error.statusCode === 404 && latestNotification) {
-          mutation.mutate(latestNotification)
-          return latestNotification
-        }
+      const id = await Governance.get().getUserLastNotification()
 
-        return null
-      }
+      return id || null
     },
     enabled: !!user && isSubscribed,
-    retry: 3,
   })
 
   const hasNewNotifications =
-    Number(userNotifications?.length) > 0 &&
-    !!lastNotificationId &&
-    latestNotification &&
-    latestNotification !== Number(lastNotificationId)
+    Number(userNotifications?.length) > 0 && latestNotification && latestNotification !== Number(lastNotificationId)
```

Comment:
Are this cast to Number necessary?

Reply:
Converted here as those values could be null or undefined and typescript didn't like to use the > operator with non-number values. But I'll remove the second one as it isn't needed.